example/glue/EnvironmentDelegate.hpp

```diff
@@ -277,6 +277,15 @@ class MM_EnvironmentDelegate
 	uintptr_t getAllocatedSizeInsideTLH() { return 0; }
 #endif /* OMR_GC_THREAD_LOCAL_HEAP */
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	/**
+	 * Reinitialize the env's language specific structures.
+	 *
+	 * @return boolean indicating whether the thread env was successfully reinitialized.
```

Comment:
```suggestion
	 * @return boolean indicating whether the thread env was successfully reinitialized
```

Reply:
All comments in these four files are fixed now.